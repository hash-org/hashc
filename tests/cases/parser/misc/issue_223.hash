// run=fail, stage=parse

wrap := (res: Result<T, (int, str)>): Result<T, IoError> => match res {
}
