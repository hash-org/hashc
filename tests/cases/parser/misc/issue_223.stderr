error: expected value assignment after type annotation within named tuple
 --> $DIR/issue_223.hash:3:36
2 |   
3 |   wrap := (res: Result<T, (int, str)>): Result<T, IoError> => match res {
  |                                      ^ here
4 |   };
  = help: consider adding a `=`
